client/my-sites/stats/wordads/index.jsx

```diff
@@ -17,8 +17,8 @@ import Intervals from 'calypso/blocks/stats-navigation/intervals';
 import DocumentHead from 'calypso/components/data/document-head';
 import EmptyContent from 'calypso/components/empty-content';
 import JetpackColophon from 'calypso/components/jetpack-colophon';
-import Main from 'calypso/components/main';
 import NavigationHeader from 'calypso/components/navigation-header';
+import Main from 'calypso/my-sites/stats/components/stats-main';
```

Comment:
Does [GoogleMyBusinessStats](https://github.com/Automattic/wp-calypso/blob/322700bb214dfcffa06cf33d88e333c576493965/client/my-sites/google-my-business/stats/index.js#L199) also need to apply this new wrapper?

Reply:
I think it's better to be consistent even we don't really have it available in Odyssey. Good catch 👍 


Addressed in 319a418ddf3